commit 898ea5cdbe78bf46ee764c99873098860d3eebbe
Author: notro <notro@tronnes.org>
Date:   Tue Jul 29 11:05:39 2014 +0200

    i2c: bcm2835: make driver available on ARCH_BCM2708
    
    Make this driver available on ARCH_BCM2708
    
    Signed-off-by: Noralf Tronnes <notro@tronnes.org>

diff --git a/drivers/i2c/busses/Kconfig b/drivers/i2c/busses/Kconfig
index 6426811..73c9cfd 100644
--- a/drivers/i2c/busses/Kconfig
+++ b/drivers/i2c/busses/Kconfig
@@ -338,7 +338,7 @@ config I2C_AU1550
 
 config I2C_BCM2835
 	tristate "Broadcom BCM2835 I2C controller"
-	depends on ARCH_BCM2835
+	depends on ARCH_BCM2835 || ARCH_BCM2708
 	help
 	  If you say yes to this option, support will be included for the
 	  BCM2835 I2C controller.
